Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
```

Comment:
I'm confused...I did not pull anything from `master`; only rebased onto the latest`rewrite-1`. Is this a problem that should be fixed in `rewrite-1`?